app/javascript/controllers/rainy_day_controller.js

```diff
@@ -25,12 +24,79 @@ export default class extends Controller {
 
 //   this.initView();
 // }
+/*******initialize method code*/
+  this.watchedAreaSlug = watchedAreaSlug // Using this context allows for the below properties to be globally accessible
+  this.options = Object.assign({}, options, {
+          'mp': {
+              'widgetSelector': '#mountainProjectWidget',
+              'defaultZoom': 11,
+              'height': 500,
+              'urlBase': 'https://www.mountainproject.com/widget'
+          },
+          'areaListElementSelector': '[data-role="rainy-day-list-option"]',
+                'activeSelectorClass': 'active',
+                'activeLabelSelector': '[data-role="active-title"]'
+    })
```

Comment:
Since we're not passing options in anymore, we can just make this a static variable on the class:

```javascript
export default class extends Controller {
  static targets = ["area1", "area2", "link"]; // we now have a this.sourceTarget or a this.sourceTargets array
  static options = {
    'mp': { ...},
    ...
  }
```

Reply:
Done!